server/controllers/work/editWork.js

```diff
@@ -0,0 +1,47 @@
+const { Work } = require('../../database/models');
+const { customError } = require('../errors');
+const { workEditValidation } = require('../../utils/validation');
+const { uploadImage, deleteImage } = require('../../utils');
+
+const editWork = async (req, res, next) => {
+  try {
+    const {
+      id, title, content,
+    } = await workEditValidation.validateAsync({ ...req.body, workImg: req.files.workImg });
+    const { providerID } = req;
+    const work = await Work.findByPk(id);
```

Comment:
why we do that this is additional request to data base which we can avoid it 
so we can read { userId, image } from request body then check 
but how can handle work doesn't exist?
to know: sequelize update method retern array has 0 if failed and one if success after ubdata we can check if zero then response user with un auth message